app/app.go

```diff
@@ -502,7 +502,8 @@ func (app *HeimdallApp) Name() string { return app.BaseApp.Name() }
 // InitChainer initializes chain
 func (app *HeimdallApp) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain {
 	var genesisState GenesisState
-	if err := json.Unmarshal(req.AppStateBytes, &genesisState); err != nil {
+
+	if err := jsoniter.ConfigFastest.Unmarshal(req.AppStateBytes, &genesisState); err != nil {
```

Comment:
Was the task only about bor and Heimdall API? Did you changed json package everywhere?

Reply:
Changed everywhere possible, except for cosmos and tendermint dependencies 
Integration tests are successful so I suppose no breaking changes have been introduced

Reply:
Ok, let's give it a try. A bit risky, but still.

Reply:
Will inform also the team about the risks so that if we detect something strange, we know how to react

Reply:
Thank you!